adit-front/src/components/TopMenu/TopMenu.js

```diff
@@ -10,13 +12,23 @@ class TopMenu extends Component {
         pic: profile,
         name: 'JIEUN',
         point: 65535,
-    } }
+    }, searchkey: null,
+    }
 
-    SignOutHandler = () => {
+    signOutHandler = () => {
         this.setState({...this.state, sign_in: true})
         window.location.assign('/signin')
     }
 
+    searchConfirmHandler = () => {
+        if(!this.state.searchkey) alert('Input your Search Words.')
+        else window.location.assign(`/adposts/search=${this.state.searchkey}`)
```

Comment:
https://stackoverflow.com/questions/47583856/window-location-href-vs-history-pushstate-which-to-use
https://stackoverflow.com/questions/10302905/location-href-property-vs-location-assign-method
위 글 참고 바람
라우터로 부터 history를 쓰는 이유는 window.location 을 통한 페이지 전환은 http request를 동반하지만 라우터의 history는 http request가 없이 "라우팅" 해 주므로 요청 안보내는 `history.push(arg)` 쓰는게 좋을듯

Reply:
Thanks for your review. Maybe I have to check all my codes on this point. It would be better to change at once after merging changes of 'createarticle' team. Thanks.